translations/tiled_de.ts

```diff
@@ -3181,12 +3186,16 @@ Bitte Format angeben.</translation>
 &lt;p&gt;Please consider supporting me as a patron. Your support would make a big difference to me, the main developer and maintainer of Tiled. I could spend less time working for money elsewhere and spend more time working on Tiled instead.&lt;/p&gt;
 &lt;p&gt;Every little bit helps. Tiled has a lot of users and if each would contribute a small donation each month I will have time to make sure Tiled keeps getting better.&lt;/p&gt;
 &lt;p&gt;&lt;i&gt;Thorbj&amp;oslash;rn Lindeijer&lt;/i&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
+&lt;h3&gt;Mit Deiner Hilfe kann ich weitermachen Tiled zu verbessern!&lt;/h3&gt;
+&lt;p&gt;Bitte überlege mich zu unterstützen. Deine Unterstützung würde einen großen Unterschied für mich als Haupt-Entwickler von Tiled machen. Ich könnte weniger Zeit damit verbringen Geld anderweitig zu verdienen und somit mehr Zeit in Tiled selbst stecken.&lt;/p&gt;
+&lt;p&gt;Jede Kleinigkeit hilft. Tiled hat sehr viele Benutzer. Würde jeder jeden Monat eine kleine Spende beitragen hätte ich Zeit sicherzugehen, dass Tiled immer besser wird.&lt;/p&gt;
+&lt;p&gt;&lt;i&gt;Thorbj&amp;oslash;rn Lindeijer&lt;/i&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
```

Comment:
I am not sure about the part above... I do not really like it overall... But not sure how to make it better without totally breaking with the english source